### PR TITLE
etcdmain: update -ca-file description

### PR DESCRIPTION
**Pull request by yichengq**
so people could deprecate old flags and use new flags much easier.

/cc @crawford @robszumski

### CODE REVIEW

File: Documentation/configuration.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 The security flags help to [build a secure etcd cluster][security].
 
 ##### -ca-file [DEPRECATED]
-+ Path to the client server TLS CA file.
++ Path to the client server TLS CA file. The recommended equivalent alternative is "--trusted-ca-file '' --client-cert-auth".
```
Comment (robszumski) [inline Documentation/configuration.md]: Can we put all of these in backticks? What's up with the empty quotes? Do we need to explain that? I don't follow what this is doing.

Comment (yichengq) [inline Documentation/configuration.md]: @robszumski If user uses '-ca-file ca.crt' before, he could change to '-trusted-ca-file ca.crt --client-cert-auth' and etcd will perform the same. I want to provide a practical way for user to stop using deprecated flags.

Comment (robszumski) [inline Documentation/configuration.md]: Can you include that specific example? I understand now, but was very confused before.

Comment (yichengq) [inline Documentation/configuration.md]: updated as you suggest

File: Documentation/configuration.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 The security flags help to [build a secure etcd cluster][security].
 
 ##### -ca-file [DEPRECATED]
-+ Path to the client server TLS CA file.
++ Path to the client server TLS CA file. '-ca-file ca.crt' could be replaced by '-trusted-ca-file ca.crt -client-cert-auth' and etcd will perform the same.
```
Comment (robszumski) [inline Documentation/configuration.md]: Can you use backticks?

Comment (yichengq) [inline Documentation/configuration.md]: sure. updated to use backtick in markdown

**Comment by yichengq** [comment]
PTAL @robszumski

**Comment by robszumski** [comment]
LGTM

**Pull request merged**